docs/addons/kubevious.md

```diff
@@ -0,0 +1,55 @@
+# Kubevious Add-on
+
+This add-on installs [Kubevious](https://github.com/kubevious/kubevious) open source Kubernetes dashboard on Amazon EKS.
+
+Kubevious provides logical grouping of application resources eliminating the need to dig through selectors and labels. 
+It also provides the ability identify potential misconfigurations using both standard and user created rules that 
+monitor the cluster
+
+## Usage
+
+```typescript
+import { App } from '@aws-cdk/core';
+import * as ssp from '@aws-quickstart/ssp-amazon-eks';
+
+const app = new App();
+
+ssp.EksBlueprint.builder()
+    .addOns(new ssp.KubeviousAddOn() )
+    .build(app, 'my-cluster');
+```
+
+## Configuration Options
+
+- `version`: Version fo the Helm Chart to be used to install Kubevious
+- `ingressEnabled`: Indicates whether to expose Kubevious using an ingress gateway. Set to false by default
+- `kubeviousServiceType`: Type of service used to expose Kubevious backend. Set to 'ClusterIP' by default
+- `mysqlRootPassword`: Default password for the root account in the MySql database. Set to 'kubevious' by default
```

Comment:
this seems to be a security issue. Is there a better way to pass this password? Can we just generate this password and store in a secret instead? Otherwise it will show up as clear text in the CFN stack.

Reply:
let's replace this with `mySqlRootPasswordSecret` and look it up using `secret-manager-utils.getSecretValue`. Ideally we could generate the secret but there is an inherent problem with CDK Kubectl provider not being able to use secrets (there is an issue on github but they won't fix). 
I would not mind it slide, but after that every partner will point to this. 
An alternative is to remove this field from the properties and let it default unless they pass it in values. (security by obscurity, not ideal, but is not a glaring hole).